common-docs/content/common-theme/front-matter/blocks/map/index.md

```diff
@@ -0,0 +1,17 @@
++++
+title = 'Map'
+time = 2
+emoji= '🗺️'
+[build]
+  render = false
+  list = 'never'
+  publishResources = false
++++
+
+A map is an ordered slice of menus. It's to show a "map" view of the syllabus, broken down into grouped modules. As with everything in this design, the point is to clarify what happens and in what order.
+
+It's just for the index page, and the front matter is
```

Comment:
what do you mean properly ordered? They can be in any order

Reply:
The prep blocks have an order - we order them in sprints, and we order the blocks within sprints. Presenting them alphabetically seems strange when there are often dependencies between them, and at least an intended partial order, as well as in practice a presented total order...

Reply:
We do order them in sprints - and they are available in those sprints in that order, so we don't need to reproduce that. It doesn't make sense to me to reproduce all the sites inside the documentation. 

To create a supra-ordering system would be an enormous undertaking. It would only reproduce something that already exists. I would also have to maintain this shadow version once things get moved around, as they inevitably would. So it's not just work now, it's work forever, as I run around after an unknown number of future people and their changes.

Overall, it seems like a door I don't want to open. I'm prepared to be wrong, but I need a good argument on this one.